code/game/gamemodes/blob/blobs/core.dm

```diff
@@ -130,6 +130,11 @@
 
 	if(!new_overmind)
 		candidates = get_candidates(ROLE_BLOB)
+
+		for(var/client/candidate in candidates)
+			if(istype(candidate.eye,/obj/item/projectile/meteor/blob/core))
```

Comment:
nope. following someone/an object locks your ghost to that mob/object. Your eye is still tied to your ghost.